GoogleUtilities/AppDelegateSwizzler/Private/GULAppDelegateSwizzler.h

```diff
@@ -55,6 +56,12 @@ typedef NSString *const GULAppDelegateInterceptorID;
  */
 + (BOOL)isAppDelegateProxyEnabled;
 
+/** Returns the current sharedApplication.
+ *
+ *  @return the current UIApplication if in an app, or nil if in extension or if it doesn't exist.
+ */
++ (nullable UIApplication *)sharedApplication;
```

Comment:
Why have you moved this to the public header? Do consumers of the AppDelegateSwizzler need this?

Reply:
@tejasd I think, I needed it in one of the versions of Messaging, but forgot to remove.